packages/seccomp-tools/PKGBUILD

```diff
@@ -0,0 +1,85 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=seccomp-tools
+pkgver=v1.6.1.r36.g728ec2f
+ver="${pkgver%%.r*}" 
+pkgrel=1
+pkgdesc="Provide powerful tools for seccomp analysis."
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-decompiler' 'blackarch-reversing')
+url="https://github.com/david942j/seccomp-tools"
+license=('MIT')
+depends=('glibc' 'ruby' 'ruby-os' 'ruby-racc')
+makedepends=('git')
+source=("git+https://github.com/david942j/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  ( set -o pipefail
+    git describe --long --tags --abbrev=7 2>/dev/null |
+      sed 's/\([^-]*-g\)/r\1/;s/-/./g' ||
+    printf "%s.%s" "$(git rev-list --count HEAD)" \
+      "$(git rev-parse --short=7 HEAD)"
+  )
+}
+
+prepare() {
+  cd $pkgname
+
+  # update gemspec/Gemfile to allow newer version of the dependencies
+  sed --in-place --regexp-extended 's|~>|>=|g' "${pkgname}.gemspec"
```

Comment:
This seems useless. The only runtime dependency with a version contrainst is https://github.com/david942j/seccomp-tools/blob/728ec2ff99643f520999fd07aa0812a594b9e3b5/seccomp-tools.gemspec#L42C3-L42C46 and `~> 1.1` is already allowing to install `1.4`.

Reply:
Fixed.